release_notes/v1.0.md

```diff
@@ -0,0 +1,450 @@
+# OPEA Release Notes v1.0
+## What’s New in OPEA v1.0
+
+- Highlights
+    - Improve the RAG performance with microservice optimization (e.g., Hugging Face TGI, vLLM) and megaservice tuning
+    - Provide the experimental LLM model training support including full fine-tuning, and parameter-efficient fine-tuning (PEFT)
+    - Improve RAG with Knowledge Graph based on Neo4j
+    - Improve VisualQnA and provide multi-modality RAG support
+    - Optimize the microservice launch and remove the dispatch overhead
+    - Enable HorizontalPodAutoscaler (HPA) for better resource management
+    - Define the metrics of RAG performance and enable accuracy evaluation for more GenAI examples
+    - Further improvement on documentation and developer experience 
+
+- Other features
+    - Enable OpenAI compatible format on new microservices
```

Comment:
This description is not accurate. Not all new added microservices can be compatible with openAI. we just enabled those microservices who can be compatible with openAI.